test/gtest/ucs/test_sys.cc

```diff
@@ -137,3 +148,16 @@ UCS_TEST_F(test_sys, memunits_to_str) {
     test_memunits(UCS_TBYTE, "1T");
     test_memunits(UCS_TBYTE * 1024, "1024T");
 }
+
+UCS_TEST_F(test_sys, cpu_cache) {
```

Comment:
This is not a solution. If for whatever reason it does not work for Arm, it has to be fixed.
Also, if this is arch specific code it should reside in arch section.

Reply:
run ```lscpu``` on Arm or Power to see what is wrong. I used it to identify available resources

Reply:
@hoopoepg maybe you are using non-standard ways to access the data. I have to do a bit more research but for now I would prefer to wait before merge.

Reply:
no problemo
as  I can see such info is available when kernel is configured to provide such info (info is read from /sys/device virtual FS). if you know better way to get it - it would be great

Reply:
@hoopoepg I'm checking with kernel folks

Reply:
Oaky I got some feedback on this. On Linux we supposed to use /sys/devices/system/cpu which  typically is up to date if run most recent linux kernel. If the kernel is not up date we would have to use some local DB to extract the info or define some default value.

Reply:
does it work on FreeBSD?